integration/common.sh

```diff
@@ -1,19 +1,19 @@
-export INTEGRATION_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-export TRILLIAN_ROOT=${INTEGRATION_DIR}/..
-export SCRIPTS_DIR=${TRILLIAN_ROOT}/scripts
-export TESTDATA=${TRILLIAN_ROOT}/testdata
+export INTEGRATION_DIR="$( cd "$( dirname "$0" )" && pwd )"
+export TRILLIAN_ROOT="${INTEGRATION_DIR}"/..
+export SCRIPTS_DIR="${TRILLIAN_ROOT}"/scripts
+export TESTDATA="${TRILLIAN_ROOT}"/testdata
 export TESTDBOPTS="-u test --password=zaphod -D test"
 export STARTUP_WAIT_SECONDS=10
 
 function runTest() {
-  echo "=== RUN   ${2}"
-  ${INTEGRATION_DIR}/$1
+  echo "=== RUN   ${1}"
+  "${INTEGRATION_DIR}"/"$2" "$3" "$4" "$5" "$6" "$7" "$8"
```

Comment:
Done for 1/2, but 3 onwards are going to be test-specific so I can't think of a sensible way to name them.